sites/game-client/map-renderer.mjs

```diff
@@ -690,6 +694,7 @@ class MapRenderer {
    * would be called to get the information for drawing.
    */
   _drawOneCharacterImage(targetCanvas, player) {
+    console.log(player);
```

Comment:
Consider not flooding the console. Use `console.debug` or not logging this information at all.